lib/sli/nest-init.sli

```diff
@@ -273,10 +273,16 @@ def
    conn_spec  dict     - dictionary with connectivity specification (see Options)
 
    Options:
-   /conn_spec - dictionary with connectivity specification, must at
-                least contain /rule
-   /syn_spec  - dictionary with synapse specification, must at least
-                contain /model
+   /conn_spec - dictionary with connectivity specification, must at least contain /rule
+
+                Valid conn_spec dictionary arguments are:
+                multapses     bool  - allow self-connections (default: true)
+                autapses      bool  - allow multiple connections between pairs
+                                      of neurons (default: true)
+                symmetric     bool  - also create connection in opposite direction to
```

Comment:
@janhahne You got autapses and multapses the wrong way around.  To avoid confusion, I would change the headline to `The following options may be given for all connection rules; not all rules support all options and many rules add rule-specific options.`